util/tracing/visualization/fedsd.py

```diff
@@ -100,6 +100,10 @@
                     help='RTI\'s lft trace file.')
 parser.add_argument('-f','--federates', nargs='+',
                     help='List of the federates\' lft trace files.')
+parser.add_argument('-s', '--start', type=str, nargs=2,
+                    help='Elapsed logical time that targets to start visualization. [time_value time_unit]')
+parser.add_argument('-e', '--end', type=str, nargs=2,
+                    help='Elapsed logical time that targets to end visualization. [time_value time_unit]')
```

Comment:
Same here "End time of visualization in elapsed logical time"

Reply:
Resolved in [14ecd1c](https://github.com/lf-lang/reactor-c/pull/320/commits/14ecd1c2e76936257403806efe3e378f347d189a).